docs/csharp/language-reference/compiler-options/advanced.md

```diff
@@ -0,0 +1,301 @@
+
+## PathMap
+
+The **PathMap** compiler option specifies how to map physical paths to source path names output by the compiler. This option maps each physical path on the machine where the compiler runs to a corresponding path that should be written in the output files. In the following example, `path1` is the full path to the source files in the current environment, and `sourcePath1` is the source path substituted for `path1` in any output files. To specify multiple mapped source paths, separate each with a semicolon.
+
+```xml
+<PathMap>path1=sourcePath1;path2=sourcePath2</PathMap>
+```
+
+The compiler writes the source path into its output for the following reasons:
+
+1. The source path is substituted for an argument when the <xref:System.Runtime.CompilerServices.CallerFilePathAttribute> is applied to an optional parameter.
+1. The source path is embedded in a PDB file.
+1. The path of the PDB file is embedded into a PE (portable executable) file.
+
+## ErrorReport
+
+This option provides a convenient way to report a C# internal compiler error to Microsoft.
+
+> [!NOTE]
+> On Windows Vista and Windows Server 2008, the error reporting settings that you make for Visual Studio do not override the settings made through Windows Error Reporting (WER). WER settings always take precedence over Visual Studio error reporting settings.
+
+```xml
+<ErrorReport>setting</ErrorReport>
+```
+
+The argument must be one of:
+
+- **none**: Reports about internal compiler errors won't be collected or sent to Microsoft.
+- **prompt**: Prompts you to send a report when you receive an internal compiler error. **prompt** is the default when you compile an application in the development environment.
+- **queue**: Queues the error report. When you sign in with administrative credentials, you can report any failures since the last time that you were logged on. You won't be prompted to send reports for failures more than once every three days. **queue** is the default when you compile an application at the command line.
+- **send**: Automatically sends reports of internal compiler errors to Microsoft. To enable this option, you must first agree to the Microsoft data collection policy. The first time that you specify `<ErrorReport>send</ErrorReport>` on a computer, a compiler message will refer you to a Web site that contains the Microsoft data collection policy.
+
+An internal compiler error (ICE) results when the compiler can't process a source code file. When an ICE occurs, the compiler doesn't produce an output file or any useful diagnostic that you can use to fix your code.
+
+By using **ErrorReport**, you can provide ICE information to the C# team. Your error reports can help improve future compiler releases. A user's ability to send reports depends on computer and user policy permissions.
```

Comment:
I don't believe the **ErrorReport** property does what you claim here. Filed <https://github.com/dotnet/docs/issues/23323> as this PR has already been merged.